features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -716,3 +724,40 @@ Feature: Gift Exchange Challenge
       And I uncheck "exchange_collection (recip)"
       And I press "Post"
     Then I should see "For recip."
+
+  Scenario: User sidebar assignments count only includes unfulfilled assignments.
+    Given the user "recip" exists and is activated
+      And I am logged in as "gifter"
+      And "gifter" has an assignment for the user "recip" in the collection "exchange_collection"
+    When I go to the assignments page for "gifter"
+    Then I should see "Assignments (1)" within "#dashboard"
+    When I fulfill my assignment
+      And I go to the assignments page for "gifter"
+      Then I should see "Assignments (0)" within "#dashboard"
```

Comment:
I wouldn't object to incorporating this with the "User fulfills their assignment and it shows on their assignments page as fulfilled" scenario. They're very interconnected and it saves us some set up.